cmd/lotus-chainwatch/storage.go

```diff
@@ -318,7 +326,47 @@ create table if not exists miner_sector_events
     
 	constraint miner_sector_events_pk
 		primary key (sector_id, event, miner_id, state_root)
-)
+);
+
+create table if not exists market_deal_proposals
+(
+    deal_id bigint not null,
+    
+    state_root text not null,
+    
+    piece_cid text not null,
+    padded_piece_size bigint not null,
+    unpadded_piece_size bigint not null,
+    verified_deal bool not null,
```

Comment:
This is chainwatch's first bool. What do you think about the convention of `is_<quality>` for bool columns and `<noun>_<verb>_at` for timestamps and timestamps_with_tz?